apps/govquests-api/rails_app/db/migrate/20241011130452_add_user_quest_id_to_user_quests.rb

```diff
@@ -0,0 +1,7 @@
+class AddUserQuestIdToUserQuests < ActiveRecord::Migration[8.0]
+  def change
+    change_table :user_quests do |t|
+      t.string :user_quest_id, null: false, index: {unique: true}
```

Comment:
why not using an "id" here? 🤔

Reply:
just the convention we established to prefix all primary keys with the table entity